src/components/withWindowDimensions.js

```diff
@@ -1,7 +1,7 @@
-/* eslint-disable react/no-unused-state */
-import React, {forwardRef, createContext} from 'react';
+import React, {forwardRef, createContext, useMemo} from 'react';
```

Comment:
I am going to let @Julesssss @roryabraham confirm if we're okay to use `useMemo` (React hooks)